components/eam/src/physics/cam/aer_rad_props.F90

```diff
@@ -794,15 +796,12 @@ subroutine volcanic_cmip_sw (state, pbuf, trop_level, ext_cmip6_sw_inv_m, tau, t
      tau_w_g(icol,ilev_tropp,:) = 0.5_r8 * ( tau_w_g(icol,ilev_tropp,:) + ext_ssa_asym(:))
      tau_w_f(icol,ilev_tropp,:) = 0.5_r8 * ( tau_w_f(icol,ilev_tropp,:) + ext_ssa_asym(:) * asym_unitless(:))
   enddo
-
-  !As it will be more efficient for FORTRAN to loop over levels and then columns, the following loops
-  !are nested keeping that in mind
-  do ipver = 1 , pver
-     do icol = 1, ncol
-        ilev_tropp = trop_level(icol) !tropopause level
+  do icol = 1, ncol
+     ilev_tropp = trop_level(icol) !tropopause level
+     do ipver = 1 , pver
```

Comment:
Is there a reason for switching the loop order? I believe the original order will be faster for Fortran. It does involve more than needed lookups in the trop_level array but the impact of ordering on timing should be much more than an arrays lookup.

Reply:
I was thinking that if Fortran considerations should not apply, reversing the loop order would allow tighter coupling of the loops.  But otherwise there is no good reason for doing so, and it can be easily reverted.

Reply:
After rebasing onto main branch, clarified in a comment that the units of extinction had been converted from 1/km to 1/m.

Reply:
You are right, switching loop order allows for tighter coupling but it may not be ideal for Fortran performance. We had to do similar tradeoffs at other places in the code. The C++ port doesn't care about the nested loops as they will have just the level loop (since it is a column model).